api/src/certification/session/domain/usecases/add-certification-candidate-to-session.js

```diff
@@ -6,6 +10,14 @@ import {
 import * as mailCheckImplementation from '../../../../shared/mail/infrastructure/services/mail-check.js';
 import { CERTIFICATION_CANDIDATES_ERRORS } from '../../../../../lib/domain/constants/certification-candidates-errors.js';
 
+/**
+ * @param {Object} params
+ * @param {deps['sessionRepository']} params.sessionRepository
```

Comment:
Pas possible comme ça?
```suggestion
 * @param {deps.sessionRepository} params.sessionRepository
```

Reply:
@aceol nope malheureusement cette syntaxe ne semble pas supportée. J'ai mis un bon moment pour trouver la bonne écriture (enfin celle qui fonctionne), j'aurais bien aimé celle avec les points aussi.